automatic/service-fabric/service-fabric.nuspec

```diff
@@ -18,7 +18,7 @@ Service Fabric is a distributed systems platform that makes it easy to package,
     <licenseUrl>https://github.com/Microsoft/service-fabric/blob/master/LICENSE</licenseUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <iconUrl>https://cdn.jsdelivr.net/gh/darinegan/chocolatey-packages@latest/icons/service-fabric.png</iconUrl>
-    <releaseNotes>[Microsoft Azure Service Fabric 10.0 Release Notes](https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_10.md)</releaseNotes>
+    <releaseNotes>[Microsoft Azure Service Fabric Release Notes](https://github.com/microsoft/service-fabric/tree/master/release_notes)</releaseNotes>
```

Comment:
Same as above; I believe this is the specific release notes for this version.
https://github.com/microsoft/service-fabric/blob/master/release_notes/Service_Fabric_ReleaseNotes_101CU3.md

Reply:
Thanks, I updated to link to MD file. I had initially pointed to release notes tree as the cadence of release notes on github isn't always updated around the same time as the Microsoft docs site with download links. Same with the releases page page https://learn.microsoft.com/en-us/azure/service-fabric/release-notes#service-fabric-101-releases